homeassistant/components/indevolt/__init__.py

```diff
@@ -0,0 +1,32 @@
+"""Home Assistant integration for INDEVOLT devices."""
+
+from __future__ import annotations
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+
+from .const import PLATFORMS
+from .coordinator import IndevoltDeviceUpdateCoordinator
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
```

Comment:
Please use `IndevoltConfigEntry` everywhere since we've defined that.


```suggestion
async def async_setup_entry(hass: HomeAssistant, entry: IndevoltConfigEntry) -> bool:
```